src/js/users/components/CreateUser.tsx

```diff
@@ -0,0 +1,54 @@
+import { DialogPortal, DialogTrigger } from "@radix-ui/react-dialog";
+import React from "react";
+import { useMutation } from "react-query";
+import { Dialog, DialogContent, DialogOverlay, DialogTitle, Icon } from "../../base";
+import { StyledButton } from "../../base/styled/StyledButton";
+import { create } from "../api";
+import { CreateUserForm } from "./CreateUserForm";
+
+type NewUserProps = {
```

Comment:
Nitpick: Props is generally reserved in the codebase for react components. I would probably name it just straight NewUser since it is the object needed to create a new user, but NewUserParams or some variant of would be fine too imo